packages/perseus/src/widgets/numeric-input/numeric-input.class.tsx

```diff
@@ -157,79 +156,34 @@ export class NumericInput
     }
 }
 
-// TODO(thomas): Currently we receive a list of lists of acceptable answer types
-// and union them down into a single set. It's worth considering whether it
-// wouldn't make more sense to have a single set of acceptable answer types for
-// a given *problem* rather than for each possible [correct/wrong] *answer*.
-// When should two answers to a problem take different answer types?
-// See D27790 for more discussion.
-export const unionAnswerForms: (
-    answerFormsList: ReadonlyArray<
-        ReadonlyArray<PerseusNumericInputAnswerForm>
-    >,
-) => ReadonlyArray<PerseusNumericInputAnswerForm> = function (answerFormsList) {
-    // Takes a list of lists of answer forms, and returns a list of the forms
-    // in each of these lists in the same order that they're listed in the
-    // `formExamples` forms from above.
-
-    // uniqueBy takes a list of elements and a function which compares whether
-    // two elements are equal, and returns a list of unique elements. This is
-    // just a helper function here, but works generally.
-    const uniqueBy = function (list, iteratee: any) {
-        // @ts-expect-error - TS2347 - Untyped function calls may not accept type arguments.
-        return list.reduce<Array<any>>((uniqueList, element) => {
-            // For each element, decide whether it's already in the list of
-            // unique items.
-            const inList = _.find(uniqueList, iteratee.bind(null, element));
-            if (inList) {
-                return uniqueList;
-            }
-            return uniqueList.concat([element]);
-        }, []);
-    };
-
-    // Pull out all of the forms from the different lists.
-    const allForms = answerFormsList.flat();
-    // Pull out the unique forms using uniqueBy.
-    const uniqueForms = uniqueBy(allForms, _.isEqual);
-    // Sort them by the order they appear in the `formExamples` list.
-    const formExampleKeys = Object.keys(NumericExampleStrings);
-    return _.sortBy(uniqueForms, (form) => {
-        return formExampleKeys.indexOf(form.name);
-    });
-};
-
 type RenderProps = {
-    answerForms: ReadonlyArray<{
-        simplify: "required" | "correct" | "enforced" | null | undefined;
-        name: "integer" | "decimal" | "proper" | "improper" | "mixed" | "pi";
-    }>;
-    labelText: string;
-    size: "normal" | "small";
+    answerForms: ReadonlyArray<PerseusNumericInputAnswerForm>;
```

Comment:
We already have this type, no need to force ourselves to maintain two locations.